src/components/SocialBar/SocialBar.tsx

```diff
@@ -0,0 +1,27 @@
+import * as React from 'react';
+import { SocialBarIcon } from './SocialBarIcon';
+
+import styles from './SocialBar.st.css';
+
+export type SocialBarTheme = 'light' | 'dark';
+
+export interface SocialBarProps {
+  theme?: SocialBarTheme;
+}
+
+interface DefaultProps {
+  theme: SocialBarTheme;
+}
+
+/** SocialBar */
+export class SocialBar extends React.Component<SocialBarProps> {
+  static Icon = SocialBarIcon;
+  static displayName = 'SocialBar';
+  static defaultProps: DefaultProps = { theme: 'light' };
+
+  render() {
+    const { theme, children, ...rest } = this.props;
+
+    return <div {...styles('root', { theme }, rest)}>{children}</div>;
```

Comment:
The way I pictured it, it would be more flexible for us if we used:
```jsx
...
return (
  <div ...>
    {React.Children.map(children, child => {
      return React.cloneElement(child, {className: classnames(styles.button, child.props.className)});
    })}
  </div>
);
```

Reply:
children of SocialBar is array of SocialBarIcon, they already have class (icon). Maybe you're talking about cloning the children of SocialBarIcon?

Reply:
cloned children